app/ui/browser/index.jsx

```diff
@@ -73,7 +73,6 @@ store.dispatch(actions.createTab());
 // that if an error occurs while we connect, we at least have some UI in place.
 ReactDOM.render(chrome, container);
 
-userAgentClient.connect().then(() => onWindowReady(true));
```

Comment:
My guess is that this will not work in production mode: I don't see where you're calling `onWindowReady` now.  Development mode always shows the window, so you probably haven't witnessed this.